docs/feature-gates.md

```diff
@@ -99,6 +101,20 @@ for more information about TopologyAwareHints.
 - `AntreaProxy` is enabled.
 - `EndpointSlice` is enabled.
 
+### LoadBalancerModeDSR
+
+`LoadBalancerModeDSR` allows users to specify the load balancer mode as DSR (Direct Server Return). The load balancer
+mode determines how external traffic destined to LoadBalancerIPs and ExternalIPs of Services is processed when it's load
+balanced across Nodes. In DSR mode, external traffic is never SNAT'd and backend Pods running on Nodes that are not the
+ingress Node can reply to clients directly, bypassing the ingress Node. Therefore, traffic in DSR mode usually has lower
+latency, higher output bandwidth, and preserved client IP. It's only meaningful to use feature when AntreaProxy is
+enabled and configured to proxy external traffic (proxyAll=true). Refer to this [link](
+antrea-proxy.md#configuring-load-balancer-mode-for-external-traffic) for more information about load balancer mode.
+
+#### Requirements for this Feature
+
+- `AntreaProxy` is enabled.
```

Comment:
I feel like there are more requirements, which we typically list here:

* `AntreaProxy` with `proxyAll` is enabled.
* IPv4 only
* Linux Nodes only
* Encap mode only

Reply:
added